eng/Build.props

```diff
@@ -123,6 +123,13 @@
         <ProjectToBuild Condition=" '$(BuildJava)' == 'true'" Include="@(JavaProjects)" Exclude="@(ProjectToExclude)" />
         <ProjectToExclude Condition=" '$(BuildJava)' != 'true'" Include="@(JavaProjects)" />
 
+        <!-- These interop test projects can only be restored and built after the shared framework is built -->
```

Comment:
There's a contradiction between this and the ability to pass `/p:BuildInteropProjects=true` in the `restore.cmd` command. Is this comment no longer correct? And, if it's incorrect, can we get rid of `$(BuildInteropProjects)` -- just always build these projects (or always build them on Windows)?

Reply:
Actually, due to the nature of the tests, we want to test with the shared framework we just built, which means we must build the interop test website/client in a separate step after everything else is built in the repo. I think we'll need to keep this current setup.